project/plugins.sbt

```diff
@@ -1 +1,3 @@
-addSbtPlugin("org.http4s" % "sbt-http4s-org" % "0.12.1")
+resolvers +=
```

Comment:
So we're good to release 0.12.2 with this?

Reply:
Yes, good to go 👍